### PR TITLE
#188 expand /status endpoint with more information

### PR DESCRIPTION
**Pull request by kostamir**

### CODE REVIEW

File: src/getCurrentVersion.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import fs from 'fs';
+
+export default async function getCurrentVersion() {
+    const packageJson = await fs.promises.readFile('./package.json', 'utf8');
```
Comment (psaulenas) [inline src/getCurrentVersion.ts:4]: You could get the version using import from `package.json`
```js
import { version } from '../package.json';
```

Then probably this separate file (function) not needed

File: src/index.ts [hunk 2]
```diff
@@ -7,14 +7,32 @@ import jwks from 'jwks-rsa';
...
+        try {
+            dbStatus = (
+                await pgClient.query({
+                    text: 'SELECT VERSION()',
+                })
+            ).rows?.[0];
+        } catch (error) {
+            console.log(error);
+        }
+        const cloudinaryStatus = await cloudinaryClient.isOk();
```
Comment (JustasBelevicius) [inline src/index.ts:28]: cloudinary isn't configured when the tests are run, therefore this throws an exception and the response is not 200

Comment (JustasBelevicius) [reply src/index.ts:28]: try/catch should fix it

File: src/getCurrentVersion.ts (hunk 1 above)
Comment (kostamir) [reply src/getCurrentVersion.ts]: Done. Thanks!

File: src/index.ts (hunk 2 above)
Comment (kostamir) [reply src/index.ts]: Done. Thanks!

**Comment by ignasposka** [comment]
Sorry, but this feature was merged from another PR.

**Pull request closed without merging**